### ISSUE TITLE
shared albums with missing photos don't download consistently on Monterey

### ISSUE THREAD

**Issue by RhetTbull** [RhetTbull/osxphotos#626]
Ok, so the behaviour is more inconsistent that I thought. I did try to repair my library *before* I read your last message, as I had read that could sort issues with regular export of photos.

I tried downloading a different album where my code had resulted in an empty folder. This time *some* of the photos/videos were downloaded (maybe 10 out of 125). I tried again, and the count went up to ~30. 

As for your second suggestion, using `--use-photokit` fails with: 

```bash
Error exporting photo (CED06217-BEC1-47CD-A539-3E36017C24FC: IMG_0107.JPG) as IMG_0107.JPG: Could not get authorizaton to use Photos: auth_status = 0
```

Add: python actually crashes when using `--use-photokit`. log attached. 
[osxphotos_crash.txt](https://github.com/RhetTbull/osxphotos/files/8017539/osxphotos_crash.txt)

_Originally posted by @moffat in https://github.com/RhetTbull/osxphotos/discussions/624#discussioncomment-2127306_

**Comment by moffat** [comment]
@RhetTbull, I have an older MBP running Catalina. I ran the same python code we discussed in #624 and it worked fine (and vastly faster!), so this does seem to be a Monterey problem.

**Comment by RhetTbull** [comment]
Thanks -- that's good feedback. I also use an old MBP running Catalina for my development...no problems! I will try to add some instrumentation to get more data on Monterey but until I eventually get access to a Monterey machine, it'll be hard to debug this.

**Comment by moffat** [comment]
Yeah, I understand. I was about to upgrade that machine but I'll hold off now to keep my shared album backups going! Thank you for all your time and this great software.
